docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -0,0 +1,127 @@
+version of *Tic-tac-toe* may have originated in ancient Egypt. Others believe that the 
+modern *Tic-tac-toe* is an evolution of a game known as *Terni Lapilli*, which was 
+popular within the Roman Empire. 
+
+*Tic-tac-toe* also has historical significance within the field of Computer Science. In
+1952, it became the basis of one of the first-ever video games when `OXO <https://en.wikipedia.org/wiki/OXO>`_ was developed by Alexander S. Douglas at the University 
+of Cambridge. More information regarding the origins and properties of *Tic-tac-toe* 
+can be found `here <https://en.wikipedia.org/wiki/Tic-tac-toe>`_.
+
+*XO* was chosen as an example transaction family for Sawtooth due to its simplicity,
+global playerbase, and straightforward implementation as a computer program. Though
+there is essentially no business application for *XO* per se, this example transaction
+family demonstrates the functionality of Sawtooth, and the code that implements it also 
+serves as a reference for building other transaction processors.
+
+How to play XO
+==============
+
+Game rules
+----------
+
+*Tic-tac-toe* is a two-player game in which players take alternating turns to mark spaces
+on a 3x3 grid. Conventionally, *player one* marks their spaces with an 'X' and *player two* marks their spaces with an 'O'; and *player one* always takes their turn first.
+
+A player wins the game by marking 3 spaces in a horizontal, vertical, or diagonal row.
+In the condition that all 9 spaces on the grid have been marked but no player has 
+achieved a winning board position, the game is considered a draw.
+
+Playing XO using the command-line interface
+-------------------------------------------
+
+In order to play *XO*, users issue commands on the *XO* command-line interface. These 
+commands are then sent through the CLI to the *XO* client for further processing.
+Instructions on how to use the CLI are detailed below:
+
+1. After starting up a validator, *XO* transaction processor, and the REST API (see `Installing and Running Sawtooth <http://intelledger.github.io/0.8/cli/xo.html>`_), 
+   each player must initialize *XO* locally using the ``xo init`` command. Initialization
+   information is stored locally in a file. This command has two options: ``--username``
+   and ``--url``.
+
+  - The ``--username`` option is the name of the player. If ``--username`` is not
+    specified, OS environment variables will be parsed to find a suitable value.
+  - The ``--url`` option should be set to the URL of the REST API. The *XO* client 
+    will send requests to this URL in order to determine the validity of each move and 
+    update the game state on the blockchain. If no URL is specified, the URL will 
+    default to ``http://127.0.0.1:8080``.
+  
+    .. note::
+
+      A user can use the ``xo reset`` command to delete local *XO* data. This includes 
+      the saved URL and username.
+
+2. A user creates a new game using the ``xo create [name]`` command.
```

Comment:
Drop the numbering. It reads fine without it and the numbering looks awkward when formatted as HTML: https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-core/job/PR-666/1/artifact/docs/build/html/app_developers_guide/intro_xo_transaction_family.html